about.html

```diff
@@ -10,7 +10,14 @@
     <link rel="stylesheet" href="./assets/css/style.css">
 </head>
 <body class="ff-lato">
-  <header class="container bg-light header-about">
+  <header class="container header header-about">
+      <div class="topbar text-light bg-dark">
```

Comment:
- [ ] [OPTIONAL] You can use ```section``` and ```article``` instead of ```<div>``` tag, for more semantic HTML and clean code